client/packages/common/src/utils/formatters/formatters.ts

```diff
@@ -13,7 +13,8 @@ export const Formatter = {
     else return null;
   },
   naiveDateTime: (date?: Date | null): string | null => {
-    if (date && isValid(date)) return format(date, "yyyy-MM-dd'T'HH:mm:ss");
+    if (date && isValid(date))
+      return format(date, "yyyy-MM-dd'T'HH:mm:ss+00:00");
```

Comment:
wasn't sure what to use for milliseconds 🤔

Reply:
the +/-hh:mm is usually for the time zone and that is what this looks like...

.mmm should work for milliseconds

However, do we want to add this to this method or do we want to have a new method `naiveDateTimeWithMs`? i.e. do we want to change existing formats?

Reply:
naiveDateTime isn't actually being used anywhere so thought it was okay to change... api needs the datetime to be in a specific format

Reply:
Ok what you need then is probably Date.toISOString(). This should be the correct format for the API.

I might be wrong, but I think this method was meant to be for formatting date times to be displayed in the UI...

Reply:
I think the csv formatters are used for displaying... as I said it's not being used anywhere, so unsure what it is there for originally~

Reply:
at one time we did format the NaiveDateTime before sending to the API. that's changed now though hasn't it?